### PR TITLE
Fixed student crud

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: controllers/student.js [hunk 1]
```diff
@@ -158,12 +173,17 @@ exports.delete_student = (req, res, next) => {
...
+            last_name: ''
+        },
+        req.body
+    ); 
     function start () {
         mysql.use('master')
             .query(
-                'SELECT * FROM student WHERE student_id = ? LIMIT 1;',
-                [req.params.id],
+                'SELECT * FROM student WHERE last_name = ? and class_id = ?;',
```
Comment (jgjadaoag) [inline controllers/student.js:185]: Bakit last name?

File: fds [hunk 2]
```diff
@@ -0,0 +1,565 @@
+[33mcommit 23fa8ccc82c8ff59f9c456c1c4622229383a11b4[m
```
Comment (jgjadaoag) [inline fds]: What is this monstrosity?

File: controllers/student.js (hunk 1 above)
Comment (ghost) [inline controllers/student.js:185]: sabi ni prime minister

File: fds (hunk 2 above)
Comment (ghost) [inline fds]: hindi ko din alam :'( nashock na lang din ako

**Pull request merged**